hbase-rest-read-write/cloudera/cod/examples/write_example.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+from requests.auth import HTTPBasicAuth
+from common import *
+import json
+import requests
+from datetime import datetime
+try:
+    from UserDict import UserDict
+    from UserDict import DictMixin
+except ImportError:
+    from collections import UserDict, OrderedDict
+    from collections import MutableMapping as DictMixin
```

Comment:
Fix collection import to fix the warning
```
DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.10 it will stop working
```